forte/data/batchers.py

```diff
@@ -144,6 +146,161 @@ def default_configs(cls) -> Dict[str, Any]:
         raise NotImplementedError
 
 
+class FixedSizeDataPackBatcherWithExtractor(ProcessingBatcher):
+    r"""This class use extractor to extract features from
+    dataset and group them into batch. In this class, more pools
+    are added. One is instance_pool, which is used to record the
```

Comment:
`instance_pool`, use the ` symbol

Reply:
I am not sure how to do this, is there a reference?